hardware_deps/Makefile

```diff
@@ -0,0 +1,14 @@
+INCLUDES = -I$(DEFINED_ERROR_PATH)
+
+ifeq ($(SIERRA_WP7702), y)
+	BUILD_PATH = sierra_wireless/wp7702
+	HARDWARE_CFLAG = CFLAGS -DSIERRA_WP7702
+endif
+
+all: device.o
+
+device.o: device.c
+	$(MAKE) -C $(BUILD_PATH)
+	$(CC) -c $(CFLAGS) $(INCLUDES) -MMD -MF device.c.d -o $@ $<
+
+-include device.c.d
```

Comment:
add new line here

Reply:
Should we also remove the `-M` command which help for check dependencies during compiling?

Reply:
I am OK witth taht at this step, but I think we may need this in the future